BraveWallet/Panels/WebpageRequestContainerView.swift

```diff
@@ -14,11 +14,28 @@ struct WebpageRequestContainerView<DismissContent: ToolbarContent>: View {
   @ObservedObject var cryptoStore: CryptoStore
   var toolbarDismissContent: DismissContent
   
+  var onDismiss: () -> Void
+  
   var body: some View {
     UIKitNavigationView {
       Group {
         if let pendingRequest = cryptoStore.pendingWebpageRequest {
-
+          switch pendingRequest {
+          case let .signMessage(requests):
+            SignatureRequestView(
+              requests: requests,
+              keyringStore: keyringStore,
+              handler: { approved, requestId in
```

Comment:
Whats the reason we do all handling of pending requests at a container level instead of at each individual view? Could we not dismiss automatically based on `cryptoStore.pendingWebpageRequest` becoming nil`?

Reply:
I will move handling of the request into the view itself (change made to Add Suggested Token PR already, others to follow).

As for dismiss, we might have to restructure the request presentation flow a bit. `WalletPanelView` currently observes changes to pending request and calls our `presentWalletWithContext` on changes; it's unaware if there is a request currently presented on screen at the moment. With latest changes in [Add Suggested Token PR](https://github.com/brave/brave-ios/pull/5187), requests will be presented sequentially as modals; we may want to match desktop behaviour and swap out views w/o dismissal.

https://user-images.githubusercontent.com/5314553/167730152-5f1986f4-0df6-49c2-82e8-fb57c192780d.mov